data/restli-csharp/datagenerator/src/test/GeneratedDataTemplate/com/linkedin/restli/datagenerator/integration/EnumInMapTest.cs

```diff
@@ -0,0 +1,36 @@
+// DO NOT EDIT - THIS FILE IS GENERATED BY restli-csharp
+// Generated from com/linkedin/restli/datagenerator/integration/MapTest.pdsc
+
+namespace com.linkedin.restli.datagenerator.integration
+{
+  
+  public class EnumInMapTest {
+    public enum Value
```

Comment:
To be consistent with our PDSC enum term (enum symbol for each enum value), can we rename `Value` as `Symbol` here?